katportalclient/client.py

```diff
@@ -71,13 +71,13 @@ def create_jwt_login_token(email, password):
     return jwt_auth_token
 
 
-class SensorSample(namedtuple('SensorSample', 'timestamp, value, status')):
+class SensorSample(namedtuple('SensorSample', 'sample_time, value, status')):
     """Class to represent all sensor samples.
 
     Fields:
-        - timestamp:  float
-            The timestamp (UNIX epoch) the sample was received by CAM.
-            timestamp value is reported with millisecond precision.
+        - sample_time:  float
+            The sample_time (UNIX epoch) the sample was received by CAM.
+            sample_time value is reported with millisecond precision.
```

Comment:
This was true for the old katstore, but not sure for new katstore - probably higher precision since we don't truncate any times.  There are a few sentences like this - suggest they are removed if they no longer apply.